browser/src/control/Control.Notebookbar.js

```diff
@@ -341,6 +342,24 @@ L.Control.Notebookbar = L.Control.extend({
 		this.reloadShortcutsBar();
 	},
 
+	disableNotebookbarButton: function(e) {
+		var toolbar = L.DomUtil.get('toolbar-up');
+		var container = toolbar.querySelector(e.buttonId);
+		var button = toolbar.querySelector(e.buttonId + '-button');
+		if (!container || !button) {
+			return;
+		}
+
+		var disabled = Boolean(e.disabled);
+		if (disabled) {
+			L.DomUtil.addClass(container, 'disabled');
```

Comment:
please do not do it manually but over JSDialog API,
in other case it might to stop working with some events at some point...

Reply:
this method: https://github.com/CollaboraOnline/online/blob/master/browser/src/control/jsdialog/Component.Toolbar.ts#L115

on this component: https://github.com/CollaboraOnline/online/blob/master/browser/src/control/Control.TopToolbar.js

Reply:
can be found in `this.map.topToolbar = JSDialog.TopToolbar(this.map);`

Reply:
ah it's for notebookbar, I confused `toolbar-up`...
above pointers are for compact toolbars, so please do something similar here:
```
this.builder.executeAction(this.parentContainer, {
	control_id: command,
	action_type: enable ? 'enable' : 'disable',
});
```

Reply:
Ok, that is the reason I could not find the disable function